### DISCUSSION TITLE
CI for other Linux distributions

### DISCUSSION THREAD

**Discussion by melezhik** [oracle/truffleruby#2778] [Show and tell]
Hi ! I am building free CI service with Ruby projects support. I have noticed your GH actions CI only is for Ubuntu. SparrowCI does out of the box builds under Alpine/Ubuntu/Debian/ArchLinux  . If you are interested here is some example build for hare programming language - https://ci.sparrowhub.io/report/1428 

So if you want to rest your project under various Linux distributions it worth a shot …

**Comment by eregon** [comment]
Thank you but we already use Docker and the internal CI to test other Linux distributions.
